crates/store/re_chunk/src/util.rs

```diff
@@ -409,3 +411,38 @@ pub fn take_array<A: ArrowArray + Clone, O: arrow2::types::Index>(
         .unwrap()
         .clone()
 }
+
+// ---
+
+use arrow2::{chunk::Chunk as ArrowChunk, datatypes::Schema as ArrowSchema};
+
+/// Concatenate multiple [`TransportChunk`]s into one.
```

Comment:
Just moved this from somewhere else -- it's very useful as an enduser as long as we're stuck with arrow2.